api/models.py

```diff
@@ -14,7 +16,7 @@ class UserProfile(models.Model):
 	profile_picture = models.ImageField(upload_to=settings.MEDIA_UPLOAD_FOLDER, blank=True)  
 	website = models.URLField(blank=True)
 	bio = models.TextField(max_length=2000, blank=True)
-	activity_status = models.CharField(max_length=50, choices=ACTIVITY_LEVEL_CHOICES, default='ACTIVE')
+	activity_status = models.CharField(max_length=50, choices=ACTIVITY_LEVEL_CHOICES, default='KITTEN')
```

Comment:
I would add some more logic here, to ask -- do you want to become a member of CodeCatz. If they say yes, then they would be assigned KITTEN activity status, and they can't change it themselves. If they say no, then nothing would change for them (sign in but no authority to do anything). Would that be okay?